consumer.go

```diff
@@ -463,6 +471,7 @@ func (w *brokerConsumer) updateSubscriptionCache(newSubscriptions []*partitionCo
 }
 
 func (w *brokerConsumer) abort(err error) {
+	w.consumer.abandonBrokerConsumer(w)
```

Comment:
Technically the subscriptionManager is a separate goroutine, so there are two, but effectively yes.